app/scripts/modules/core/src/presentation/Markdown.tsx

```diff
@@ -32,9 +34,9 @@ export function Markdown(props: IMarkdownProps) {
   }
 
   const restProps = rest as React.DOMAttributes<any>;
-  const parsed = parser.parse(message.toString());
+  const parsed = parser.parse(message.trim().toString());
```

Comment:
<img width="301" alt="Screen Shot 2019-12-19 at 2 39 31 PM" src="https://user-images.githubusercontent.com/73450/71215879-ca5b9280-226d-11ea-9041-2b5500046fcd.png">

Fact: if a string starts with four or more spaces, markdown puts it in a `<pre>` tag, which is...not what we want literally ever I think

Reply:
I don't feel particularly good about this.  If we claim to support markdown we should not alter the semantics of the input text.

Reply:
I don't feel good about this.  If we claim to support markdown, we shouldn't alter markdown semantics